cmd/minishift/cmd/start.go

```diff
@@ -255,13 +270,15 @@ func determineInsecureRegistry(key string) []string {
 	return append(s, defaultInsecureRegistry)
 }
 
-func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
+func startHost(libMachineClient *libmachine.Client, vmDriver string) *host.Host {
```

Comment:
was previously suggested to remove the Viper call and pass it to the function...

actually, it would be nicer if the whole config struct gets extracted, as some of the information in this can be used for the precheck or start check improvements.